src/test/java/com/example/bookstoreweb/repository/BookRepositoryTest.java

```diff
@@ -0,0 +1,47 @@
+package com.example.bookstoreweb.repository;
+
+import com.example.bookstoreweb.model.Book;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.jdbc.Sql;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class BookRepositoryTest {
+    private static final String SCRIPT_FOR_ADD_DATA_IN_DB =
+            "classpath:database/add-book-to-books-table.sql";
+    private static final String SCRIPT_FOR_REMOVE_DATA_IN_DB =
+            "classpath:database/remove-data-from-all-tables.sql";
+    @Autowired
+    private BookRepository bookRepository;
+
+    @Test
+    @DisplayName("Find all books by category id")
+    @Sql(scripts = SCRIPT_FOR_ADD_DATA_IN_DB,
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = SCRIPT_FOR_REMOVE_DATA_IN_DB,
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+    void findAllByCategoryId_ValidId_ReturnOneBook() {
```

Comment:
![image](https://github.com/mhorianin/book-store-web/assets/49767049/d1aacb66-5316-4e64-8251-08121b7da60e)
sorry for not being clear enough, I meant extract to a class level